packages/pwa/app/utils/utils.js

```diff
@@ -162,3 +164,107 @@ export const capitalize = (text) => {
  * @returns {object} - url object from the pwa-kit-config.json file
  */
 export const getUrlConfig = () => pwaKitConfig?.app?.url
+/**
+ * A util to return current multi site configuration
+ * @returns {object} - a list of site objects from the pwa-kit-config.json file
+ */
+export const getSitesConfig = () => pwaKitConfig?.app?.sites
+/**
+ * A util to return current defaultSiteId
+ * @returns {string} - default site Id string from pwa-kit-config file
+ */
+export const getDefaultSiteId = () => pwaKitConfig?.app?.defaultSiteId
```

Comment:
Have you thought of creating a more generic utility, I don't see this pattern holding up after we add more and more configuration values to it. Options could be something like `getConfig([key])` where key is optional, maybe we pass in a path to a key. Either way, having so many utilities might get out of hand.

Reply:
That makes sense to me 👍